sdk/test/Services/BedrockRuntime/IntegrationTests/BedrockRuntimeEventStreamTests.cs

```diff
@@ -25,48 +25,7 @@ namespace AWSSDK_DotNet.IntegrationTests.Tests
     [TestClass]
     public class BedrockRuntimeEventStreamTests : TestBase<AmazonBedrockRuntimeClient>
     {
-#if BCL35
```

Comment:
Not sure why these didn't show up before (my guess is there's a lot of generated files and my local search is struggling a bit).